### DISCUSSION TITLE
Why no const assertion on the palette theme object?

### DISCUSSION THREAD

**Discussion by RhyG** [infinitered/ignite#2326] [Q&A] [answered]
G'day, just curious why there's no const assertion on the palette object? i.e.
```typescript
const palette = {
  neutral100: '#FFFFFF',
  neutral200: '#F4F2F1',
 ...
} as const;
```
This would mean string literals in the type:
![Screenshot 2022-12-03 at 12 16 38](https://user-images.githubusercontent.com/47733004/205417735-ae344168-f4fb-4782-9999-8aed489f1a7c.png)
Rather than what is currently just `string`:
![Screenshot 2022-12-03 at 12 17 17](https://user-images.githubusercontent.com/47733004/205417762-0d0f7ade-5a07-459c-ba8a-2a8218d5eba9.png)
And in some cases this also gives better autocomplete.

Not sure if this is for a specific reason, so keen to fill that knowledge gap for myself if that's the case.

Thanks heaps, love the repo!

**Comment by robinheinze** [comment, marked as answer]
Nope, no particular reason! Feel free to add yourself, or file an issue :)

**Reply by RhyG** [reply]
Awesome will do, thanks heaps!
